internal/pkg/template/input/step.go

```diff
@@ -0,0 +1,18 @@
+package input
+
+type StepsGroups []StepsGroup
+
+type StepsGroup struct {
+	Description string `json:"description" validate:"max=80"`
+	Required    string `json:"required" validate:"oneof=all atLeastOne exactOne zeroOrOne optional"`
+	Steps       []Step `json:"steps" validate:"min=1,dive"`
+}
+
+type Step struct {
+	Icon              string `json:"icon"`
+	Name              string `json:"name" validate:"required,max=20"`
+	Description       string `json:"description" validate:"max=40"`
+	DialogName        string `json:"dialogName,omitempty" validate:"max=20"`
+	DialogDescription string `json:"dialogDescription,omitempty" validate:"max=200"`
+	Inputs            Inputs `json:"inputs" validate:"required,dive"`
```

Comment:
A sorry, az na druhy pohlad som si vsimol ... 
Moze byt `step`, ktory je uplne prazdny, a nema ziadny `input`, ... bude tam iba pre ilustraciu, vid `transformation` v design dokumente (takto to vymysle DESI team).

Teda tu by malo asi byt `omitempty,dive`.

Reply:
![image](https://user-images.githubusercontent.com/19371734/159892950-946f5310-d349-4c6b-ab5c-29ccb48dba92.png)

Reply:
Jo ok. Zrovna jsem nad tím přemýšlel. 👍